src/sentry/static/sentry/app/components/organizations/timeRangeSelector/index.jsx

```diff
@@ -186,30 +182,34 @@ const SelectorItem = styled(
       onClick: PropTypes.func.isRequired,
       value: PropTypes.string,
       label: PropTypes.node,
-      selected: PropTypes.bool,
     };
 
     handleClick = e => {
       let {onClick, value} = this.props;
       onClick(value, e);
     };
     render() {
-      let {className, label, selected} = this.props;
+      let {className, label} = this.props;
       return (
         <Flex className={className} onClick={this.handleClick}>
           <Label>{label}</Label>
-
-          {selected && <input style={{margin: 0}} type="checkbox" checked disabled />}
         </Flex>
       );
     }
   }
 )`
   cursor: pointer;
   white-space: nowrap;
-  padding: ${space(1)} ${space(2)};
+  padding: ${space(1)};
   align-items: center;
-  border-bottom: 1px solid ${p => p.theme.borderLight};
+  flex: 1;
+  background-color: ${p => (p.selected ? p.theme.offWhite : 'transpatent')};
+  font-weight: ${p => (p.selected ? 'bold' : 'normal')};
+  border-bottom: 1px solid ${p => (p.last ? 'transparent' : p.theme.borderLight)};
```

Comment:
Would `:last-child` work?

Reply:
last child is kinda poorly implemented and I would prefer we avoid it unless there is a situation where we truly want to apply a style to anything appended to the end of the parent _no matter what it is_ (but in my experience this situation never occurs)